de-lib/src/main/java/org/iplantc/de/tools/client/views/manage/ToolInfoViewImpl.java

```diff
@@ -70,6 +71,32 @@ private void buildToolInfoView(Tool tool) {
                           new AbstractHtmlLayoutContainer.HtmlData(".cell5"));
         htmlContainer.add(new Label(tool.getDescription()),
                           new AbstractHtmlLayoutContainer.HtmlData(".cell7"));
+        htmlContainer.add(new Label(appearance.restrictions()),
+                          new AbstractHtmlLayoutContainer.HtmlData(".cell9"));
+        htmlContainer.add(new Label(
+                                  appearance.memLimit() + ((tool.getContainer().getMemoryLimit() == null) ?
+                                                           appearance.notApplicable() :
+                                                           tool.getContainer().getMemoryLimit())),
+                          new AbstractHtmlLayoutContainer.HtmlData(".cell11"));
+        htmlContainer.add(new Label(
+                                  appearance.pidsLimit() + ((tool.getContainer().getPidsLimit() == null) ?
+                                                            appearance.notApplicable() :
+                                                            tool.getContainer().getPidsLimit())),
+                          new AbstractHtmlLayoutContainer.HtmlData(".cell12"));
+        htmlContainer.add(new Label(appearance.timeLimit().toString() + ((tool.getTimeLimit() == null
+                                                                          || tool.getTimeLimit() == 0) ?
+                                                                         appearance.notApplicable() :
+                                                                         tool.getTimeLimit())),
+                          new AbstractHtmlLayoutContainer.HtmlData(".cell13"));
+        htmlContainer.add(new Label(
+                                  appearance.networkingMode() + ((Strings.isNullOrEmpty(tool.getContainer()
+                                                                                            .getNetworkMode())
+                                                                  || tool.getContainer()
+                                                                         .getNetworkMode()
+                                                                         .equalsIgnoreCase(appearance.bridge())) ?
+                                                                 appearance.enabled() :
+                                                                 appearance.disabled())),
```

Comment:
This might just be a preference but for these 5 new Labels I think it makes more sense to have the appearance return the entire string result. 
So something like `htmlContainer.add(new Label(appearance.networkingMode(tool.getContainer.getNetworkMode()))`. 
This way, the view isn't building what the final string will be, that responsibility goes to the appearance and everything can be found in the appearance.

Reply:
Right! But I was thinking those strings are different. Appearance tells me what to display and view can decide how to display.

Reply:
Sure, I guess in my mind I see it as the view chooses where the Label is in the layout (the `how to display`), and then the appearance decides `what to display` by deciding in full what the text will be. So if the text needed to change, I would expect there to be a change in the appearance not the view. Again, it might just be my own personal preference. PR is good to merge 👍

Reply:
Also I think it's useful if the appearance can also decide where the value goes in the label string.
Right now, the code is assuming `labelPrefix + value`, but if we pass the value to the appearance, then it can decide where to position it in the future: `appearance.networkingMode(networkingModeValue)`.

I also think this code would look cleaner if the label-value-logic was refactored out of the `htmlContainer.add(new Label(...))` calls:
```java
String networkMode = tool.getContainer().getNetworkMode();
String networkingModeValue = (Strings.isNullOrEmpty(networkMode) || networkMode.equalsIgnoreCase(appearance.bridge())) ?
    appearance.enabled() :
    appearance.disabled());
```

Reply:
I guess I was going for concise 1 liners. But coming back to appearance,  I think there is a basic assumption that `label` and `appearance` is going to be in one line. But that can change as well. For example
>Networking:
>   Disabled

In this case, I just have to change the underlying html template and not have rewrite the appearance. May be I could move the logic of what should be displayed for value to appearance. That way I could keep still keep the `label` and `value` separate in appearance but code is little bit cleaner in view.

Reply:
> In this case, I just have to change the underlying html template

👍 

Speaking of concise 1-liners:
```java
String networkingModeValue = (appearance.bridge().equalsIgnoreCase(tool.getContainer().getNetworkMode()) ? appearance.enabled() : appearance.disabled());
```
I have no strong opinion on where this logic goes.